weblab_cg/tests/test_chaste_cg.py

```diff
@@ -90,3 +93,90 @@ def test_missing_capacitance(self, tmp_path):
                                                 'pandit_model_2001_epi_old_no_capacitance')
         assert str(error.value) == \
             'Membrane capacitance is required to be able to apply conversion to stimulus current!'
+
+    @pytest.mark.chaste
+    def test_script_help(self, capsys):
+        testargs = ["chaste_codegen", "-h"]
+        with mock.patch.object(sys, 'argv', testargs):
+            try:
+                chaste_codegen()
+            except SystemExit:
+                pass  # We expect this to print usage and exit
+            captured = capsys.readouterr()
+            # compare to expected
+            output = str(captured.out)
+            expected = open(os.path.join(cg.DATA_DIR, 'tests', 'console_sctipt_help.txt'), 'r').read()
+            assert output == expected
+
+    @pytest.mark.chaste
+    def test_script_version(self, capsys):
+        testargs = ["chaste_codegen", "--version"]
+        with mock.patch.object(sys, 'argv', testargs):
+            try:
+                chaste_codegen()
+            except SystemExit:
+                pass  # We expect this to print usage and exit
+            captured = capsys.readouterr()
+            # compare to expected
+            output = str(captured.out)
+            assert output == 'chaste_codegen ' + cg.__version__ + '\n'
+
+    @pytest.mark.chaste
+    def test_script_convert(self, capsys, tmp_path):
+        tmp_path = str(tmp_path)
+        model_name = 'hodgkin_huxley_squid_axon_model_1952_modified'
+        model_file = os.path.join(cg.DATA_DIR, 'tests', 'cellml', model_name + '.cellml')
+        assert os.path.isfile(model_file)
+        testargs = ["chaste_codegen", model_file]
+        # Call commandline script
+        savedPath = str(os.getcwd())
+        os.chdir(tmp_path)
```

Comment:
Rather than having to change into a temporary folder, the conversion script should perhaps gain the option to specify an output folder?

Reply:
it already has (-o) but this test is testing what happens when -o is not specified.